primitives/ls.yaml

```diff
@@ -1175,16 +1175,16 @@ definitions:
   #INTEL - AVX512
   - target_extension: "avx512"
     ctype: [ "uint32_t", "int32_t", "uint64_t", "int64_t" ]
-    lscpu_flags: [ "avx512f" ]
+    lscpu_flags: [ "avx512f", "avx512vbmi2" ]
```

Comment:
Why do we need avx512vbmi2? This is only needed for 1- and 2-byte data types ([1]). Are there any compiler errors without the avx512vbmi2 flag?

[1] https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm512_mask_compressstoreu_epi&ig_expand=1481,1484,1487,1478

Reply:
Removed.